README.md

```diff
@@ -177,6 +177,18 @@ In alphabetic order.
     </td>
   </tr>
 
+<tr>
+   <td>
+     <img width="300px" src="https://numfocus.org/wp-content/uploads/2016/07/shogun-logo-300.png">
```

Comment:
Can you add the image to the repo directly, please. I had problems with missing images due to NumFOCUS changing links in the past. It's more stable to link against our own images.